client/sharkey_client.go

```diff
@@ -237,3 +216,21 @@ func buildConfig(caBundlePath string) (*tls.Config, error) {
 		MinVersion: tls.VersionTLS12,
 	}, nil
 }
+
+func (c *context) shellOut(command []string) {
+	if c.conf.Sudo {
+		command = append([]string{"/usr/bin/sudo"}, command...)
```

Comment:
Would it work to have `sudo` here instead of `/usr/bin/sudo` to support it being at different paths?